src/test/operators/aggregate_test.cpp

```diff
@@ -177,6 +178,52 @@ TYPED_TEST(OperatorsAggregateTest, CannotStandardDeviationSampleStringColumns) {
   EXPECT_THROW(aggregate->execute(), std::logic_error);
 }
 
+// The ANY aggregation is a special case which is used to obtain "any value" of a group of which we know that each
+// value in this group is the same (for most cases, the group will have a size of one). This can be the case, when
+// the aggragated column is functionally dependent on the group-by columns. In debug mode, we DebugAssert that each
+// value is the same within the group.
+TYPED_TEST(OperatorsAggregateTest, CannotUseAnyOnNonDependentColumn) {
```

Comment:
"CannotUseAnyOnNonDependentColumn" - this is not really enforced anywhere is it? Isn't this test more a "ANY complains if not all values are equal"?

Reply:
Right. `FailAnyOnNonDependentColumn` ?